atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/ColumnValueDescription.java

```diff
@@ -216,10 +241,22 @@ public Class<?> getJavaObjectTypeClass() {
         return type.getJavaObjectClass();
     }
 
+    /**
+     * Legacy persister method; will fail for reusable persisters.
+     *
+     * @deprecated Use `getReusablePersister` instead.
+     */
+    @Deprecated
     public Persister<?> getPersister() {
```

Comment:
Not entirely sure if we need to do this, but `ColumnValueDescription` and this method are both `public`, so adding this in case people are relying on the types here (do people implement meta-libraries?).